yt_dlp/extractor/arte.py

```diff
@@ -181,13 +197,20 @@ def _real_extract(self, url):
                 self.report_warning(f'Skipping stream with unknown protocol {stream["protocol"]}')
 
             # TODO: chapters from stream['segments']?
-            # The JS also looks for chapters in config['data']['attributes']['chapters'],
-            # but I am yet to find a video having those
 
         formats.extend(secondary_formats)
         self._remove_duplicate_formats(formats)
 
         metadata = config['data']['attributes']['metadata']
+        chapters = []
+        for chapter in traverse_obj(config, ('data', 'attributes', 'chapters', 'elements'),
+                                    expected_type=list) or []:
```

Comment:
branching with traverse_obj (`...`) will always return a list
```suggestion
        for chapter in traverse_obj(config, ('data', 'attributes', 'chapters', 'elements', ...)):
```